packages/react-vapor/src/components/dropdownSearch/MultiSelectDropdownSearch/DraggableSelectedOption.tsx

```diff
@@ -93,7 +94,7 @@ export class DraggableSelectedOption extends React.PureComponent<IDraggableSelec
                 <div className="selected-option-wrapper" style={{opacity}}>
                     <SelectedOption
                         {..._.omit(this.props, DraggableSelectedOptionPropsToOmit)}
-                        label={this.props.isDragging ? null : this.props.label}
+                        label={this.props.isDragging ? null : this.props.hoverTooltipValue || this.props.label}

```

Comment:
`this.props.isDragging ? null : this.props.hoverTooltipValue ?? this.props.label` 
The `||`(or) operator certainly works here but the nullish coalesing operator is more appropriate for this use case.